nnpdf_data/theory.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module provides an unique source and definition for all the possible parameters
+that a theory card can contain. 
+It also implement some utilities function to manage the theory cards. 
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class _TheoryCard:
```

Comment:
```suggestion
class TheoryCard:
```
we want to expose this